assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -152,6 +152,97 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
     processInitStateVariable(currentStateSnapshot, simLife.domain);
   }, []);
 
+  const processMutateStateVariable = (currentStateSnapshot: any, domain = 'stage') => {
```

Comment:
I have just separated the init state and mutate state variable processing. It was causing some issues in 'bindTo' operator functionality